examples/chart/teleport-demo/docker/cloudflare-agent/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+set -e
+VERSION=3.2.0
+if [[ "$1" != "" ]]; then
+    VERSION=$1
+    shift
+fi
+docker pull quay.io/gravitational/debian-grande:latest
```

Comment:
docker build has special flag `      --pull                    Always attempt to pull a newer version of the image` which you can use and skip this step.